<!doctype html>
<html lang="us">
<head>
	<meta charset="utf-8">
<meta property="og:title" content="Elliott Lab">
<meta property="og:description" content="Machine learning with beauty and care.">
<meta property="og:image" content="http://elliottlab.ca/out4.png">
<meta property="og:image:secure_url" content="https://elliottlab.ca/out4.png">
<meta property="og:image:type" content="image/png" />
<meta property="og:image:width" content="250" />
<meta property="og:image:height" content="273" />
<meta property="og:url" content="https://elliottlab.ca/">



	<title>elliottlab.ca</title>
	<link href="jquery-ui.css" rel="stylesheet">
	<script src="external/jquery/jquery.js"></script>
	<script src="jquery-ui.js"></script>
<!--script src="polyfill.min.js?features=es6"></script-->
<script id="MathJax-script" async src="MathJax/es5/tex-mml-chtml.js"></script>

	<style>
	body {
		font-family: "Trebuchet MS", sans-serif;
		background-color: #222222; 
		color: #e0dee2;
		margin: 0px;
		border: 0px;
		padding: 0px;
	}

	div.toc {
		padding-top: 25px;
		padding-bottom: 0px;
		padding-left: 5px;
	}

	html { overflow-y: scroll; }

	div.main1 {
		background-color: #3d3639;
		margin-top: 0px;
		margin-bottom: 0px;
		margin-left: 0px;
		margin-right: 0px;
		padding-left: 0px;
		padding-right: 0px;
		padding-top: 0px;
		padding-bottom: 25px;
		border: 0px;
	}

	div.main2 {
		margin-top: 0px;
		margin-bottom: 0px;
		margin-left: 50px;
		margin-right: 50px;
		padding-top: 0px;
		padding-left: 0px;
		padding-right: 0px;
		padding-bottom: 5px;
		border: 0px;
	}


	div.middle1 {
		background-color: #3d3639;
		margin-top: 0px;
		margin-bottom: 0px;
		margin-left: 0px;
		margin-right: 0px;
		padding-left: 0px;
		padding-right: 0px;
		padding-top: 0px;
		padding-bottom: 0px;
		border: 0px;
	}

	span.middle2 {
		margin-top: 0px;
		margin-bottom: 0px;
		margin-left: 50px;
		margin-right: 50px;
		padding-top: 0px;
		padding-left: 0px;
		padding-right: 0px;
		padding-bottom: 0px;
		border: 0px;
	}

	div.middle2 {
		margin-top: 0px;
		margin-bottom: 0px;
		margin-left: 50px;
		margin-right: 50px;
		padding-top: 0px;
		padding-left: 0px;
		padding-right: 0px;
		padding-bottom: 0px;
		border: 0px;
	}

	li {
		list-style-position: inside;
text-indent: -1em;
margin-bottom: 35px;
	}

	div.top {
		margin-top: 0px;
		margin-left: 50px;
		margin-right: 50px;
		margin-bottom: 0px;
	}


    u {
        text-decoration: underline;
        -webkit-text-decoration-color: #9f9f9f;
        text-decoration-color: #9f9f9f;
    }

	h2 {
		padding-top: 20px;
	}

	hr {
    border: none;
    height: 5px;
    color: #58525e; /* 58525e old IE e0dee2 706f71*/
    background-color: #58525e; /* Modern Browsers */
}
}

	.demoHeaders {
		margin-top: 2em;
	}
	#dialog-link {
		padding: .4em 1em .4em 20px;
		text-decoration: none;
		position: relative;
	}
	#dialog-link span.ui-icon {
		margin: 0 5px 0 0;
		position: absolute;
		left: .2em;
		top: 50%;
		margin-top: -8px;
	}
	#icons {
		margin: 0;
		padding: 0;
	}
	#icons li {
		margin: 2px;
		position: relative;
		padding: 4px 0;
		cursor: pointer;
		float: left;
		list-style: none;
	}
	#icons span.ui-icon {
		float: left;
		margin: 0 4px;
	}
	.fakewindowcontain .ui-widget-overlay {
		position: absolute;
	}
	select {
		width: 200px;
	}

	li {

		line-height: 1.5;
	}
	p.content {
		line-height: 1.5;
		padding-bottom: 5px;
	}

	div.content {

		  display: flex;
  flex-flow: column;
  height: 100%;
	}


span.title {
	color: #bbb4c2; /*#ae90ff; /*#f92472; /* f92472 c3245e 259e00 */
	font-weight: bold;
}

span.option {
	color: #bbb4c2; /*#54d992; /*#67d8ef; /* a6e22c 26816e 259e00   9689a4 */
	font-weight: bold;
}

span.command {
	color: #bbb4c2; /*#67d8ef; /* ac80ff #71266e; */
	font-weight: bold;
}
	ol.content {
		line-height: 1.25;
	}

	ul.content {
		line-height: 1.25;
		padding-bottom: 5px;
	}

  .math {font-size: 67%; color: #888888;}

div.manual1 {
	background-color: #292328; #58525e; /*#e0dee2;*/
	color: #9689a4;
	/*color: #000000;*/
height: 100%;
	padding: 0px;
	border: 0px;
	margin: 0px;
	float:left;
	overflow:auto; 
}

@font-face {
    font-family: 'DroidSansMono';
    src: url('DroidSansMono.ttf');
}

div.manual2 {
	font-family: DroidSansMono;
	margin: 50px;
	height: 100%;
	float:left;
		line-height: 2;
	overflow:auto; 
}

a:link {
  text-decoration: none;
  color: #9689a4;
}

a:visited {
  text-decoration: none;
  color: #9689a4;
}

a:hover {
  text-decoration: none;
  color: #bbb4c2;
}


a:active {
  text-decoration: none;
  color: #bbb4c2;
}

div.header {
	width: 100%;
	margin-left: 0px;
	margin-right: 0px;
	margin-top: 0px;
	margin-bottom: 0px;
	padding: 0px;
	border: 0px;
	top: 0px;
	left: 0px;
	background-color: #58525e;
}


div.footer {
	width: 100%;
	margin-left: 0px;
	margin-right: 0px;
	margin-top: 5px;
	margin-bottom: 0px;
	padding: 0px;
	border: 0px;
	left: 0px;
	background-color: #3d3639;
}

		.title {
			color: bab4c1 !important;
		}



	div.bottom {
		margin-top: 0px;
		margin-left: 50px;
		margin-right: 50px;
		margin-bottom: 0px;
	}



	</style>
</head>
<body>
	<div style="background-color: black; width:100%; height:30px; float:left;">
	<a href="https://www.sfu.ca"><img src="black.png" style="float:left; width:60px; margin-right:20px; border:0px;"></a>
</div>
	<div style="background-color: #58525e;">
<div class="header">
	<big><center>
	<div style="line-height: 2.0; padding-top:15px; padding-left:15px; padding-right: 15px;">
	<a href="index.html">Home</a> &nbsp;&nbsp;
	<a href="people.html">People</a> &nbsp;&nbsp;
	Papers &nbsp;&nbsp;
	<a href="software.html">Software</a> &nbsp;&nbsp;
	<a href="data.html">Data</a> &nbsp;&nbsp;
	<a href="jc.html">JC</a> &nbsp;&nbsp;
	<a href="events.html">Events</a> &nbsp;&nbsp;
	<a href="teaching.html">Teaching</a>
</div>
</center></big>
<span>&nbsp;
	<div class="top">
<h1><i>elliottlab</i></h1>
</div>&nbsp;
</span>
</div> 
<div>
<div style="right:0px; top:98px; position: absolute;">
<div style="margin: 0px; border-left: 0px; position: static; right: 0px; border-right: 0px; border-bottom: 0px; border-style: solid; border-color: #58525e;">
	<table><tr><td>
	<div style="position:relative; top:10px; margin-top:25px; margin-left:20px;">
	 <i><big>Machine learning with beauty and care.</big></i>
	</div></td><td>
	<img style="top:4px; left: -5px; position: relative;" src="out4.png" width=250 height=273></td></tr></table>
</div></div>
</div>
<div class="middle1">
<span class="middle2">
<div class="middle2" style="margin-top:30px;">
	<div class="toc">

</div>


<h2><a id="introduction"></a>Papers</h2>
<br>
<br>
<div class="content" ><ul>

	<li>
		<u>E. Sidrow</u>, A. Bouchard-Côté and <u>L.T. Elliott</u>. <em><a href="https://arxiv.org/abs/2502.15110">Variational phylogenetic inference with products over bipartitions</a>.</em> 2025. arxiv preprint 2502.15110. <em><a href="https://github.com/EvanSidrow/VIPR">Code</a></em>
	</li>

	<li>
		<u>J. Min</u>, <u>O. Vishnyakova</u>, A. Brooks-Wilson and <u>L.T. Elliott</u>. <em><a href="https://www.liebertpub.com/doi/full/10.1089/cmb.2024.0563?casa_token=txp1AXoyq0IAAAAA%3APNVVxDbgVHhXNo5-gLiY_n4y81aK2R1iCcyIBKlZ8yj4po30xlh4KlF76zCK4Ww_Dsw7hya52tYvOAAN">A joint Bayesian model for change-points and heteroskedasticity applied to the Canadian Longitudinal Study on Aging</a>.</em> Journal of Computational Biology. 2025. 32(4). p374-93. <em><a href="https://www.liebertpub.com/doi/suppl/10.1089/cmb.2024.0563/suppl_file/cmb.2024.0563_supplementary_data.pdf">Supplementary material 1</a> & <a href="https://www.liebertpub.com/doi/suppl/10.1089/cmb.2024.0563/suppl_file/cmb.2024.0563_supplementary_table_s1.pdf">2</a></em>
	</li>
	
<li>
	G. Morgan, C.Y.J. Fung, A.C. Gingras, K. Colwill, L. Briollais, E. Frangione, D. Wolday, F. Qi, A. Pasculescu, M. Delgado-Brand, G. Mailhot, T. Tursun, S. Arnoldo, E. Bearrs, A. Binnie, B. Borgundvaag, S. Casalino, S. Chowdhary, M. Dagher, L. Devine, <u>L.T. Elliott</u>, S.M. Friedman, Z. Khan, E. Lapadula, G. MacDonald, T. Mazzulli, S.L. McLeod, C. Mighton, K. Nirmalanathan, D. Richardson, S. Stern, A. Taher, J. Young, J. Lerner-Ellis and J. Taher.
	<em><a href = "https://www.sciencedirect.com/science/article/pii/S000991202400153X">Characterizing the SARS-CoV-2 antibody response and associations with patient factors: Serological profiling of participants enrolled in the GENCOV study</a>.</em> 2024. Clinical Biochemistry. 135. <em><a href="https://www.sciencedirect.com/science/article/pii/S000991202400153X/pdfft?md5=f1f7538d03026caf3a87a717c732f09f&pid=1-s2.0-S000991202400153X-main.pdf">Manuscript</a></em>
</li>
	
	 <li>
                
		 <u>M.R.P. Parker</u>, J. Cao, L.L.E. Cowen, <u>L.T. Elliott</u> and J. Ma.
                    <em><a href="https://www.projecteuclid.org/journals/annals-of-applied-statistics/volume-18/issue-4/Multisite-disease-analytics-with-applications-to-estimating-COVID-19-undetected/10.1214/24-AOAS1915.short">Multi-site disease analytics with applications to estimating COVID-19 undetected cases in Canada</em></a>.
		 Annals of Applied Statistics. 2024. 18(4). p2928-49. <i><a href="https://www.projecteuclid.org/journals/supplementalcontent/10.1214/24-AOAS1915/aoas1915suppb.pdf">Supplementary material</a>, <a href="https://www.projecteuclid.org/journals/supplementalcontent/10.1214/24-AOAS1915/aoas1915suppa.zip">Code</a> & <a href="https://www.medrxiv.org/content/10.1101/2022.07.11.22277508v1">Preprint version</a></i>
         
            </li>
	
	<li>
		J. Bambi, H. Sadri, K. Moselle, E. Chang, Y. Santoso, J. Howie,
A. Rudnick, <u>L.T. Elliott</u> and A. Kuo. <em><a href="https://www.mdpi.com/2673-7426/4/3/103">
Approaches to extracting patterns of service utilization for patients with complex conditions: Graph community detection vs. natural language processing clustering</a>.</em> 2024. BioMedInformatics. 4(3). p1884-900
	</li>
<li>
<u>M.R.P. Parker</u>, J. Cao, L.L.E. Cowen and <u>L.T. Elliott</u>.
<em><a class="title" href="https://link.springer.com/article/10.1007/s13253-024-00618-w">
Faster asymptotic solutions for N-mixtures on large populations</a>.</em>
2024.
Journal of Agricultural, Biological and Environmental Statistics. <i><a href="Parker2024.pdf">Manuscript</a>, <a href="https://static-content.springer.com/esm/art%3A10.1007%2Fs13253-024-00618-w/MediaObjects/13253_2024_618_MOESM2_ESM.zip">Code</a> & 
<a href="https://static-content.springer.com/esm/art%3A10.1007%2Fs13253-024-00618-w/MediaObjects/13253_2024_618_MOESM1_ESM.pdf">Supplementary material</a></i>

</li>
	
<li>
J. Manuello,
<u>J. Min</u>,
P. McCarthy,
F. Alfaro-Almagro,
S. Lee, S. Smith,
<u>L.T. Elliott</u>,
A.M. Winkler
and G. Douaud.
<em><a href="https://www.nature.com/articles/s41467-024-46344-2/">
The effects of genetic and modifiable risk factors on brain regions vulnerable to ageing and disease</a>.</em>
2024.
Nature Communications.
15(2576). <i><a href="https://www.nature.com/articles/s41467-024-46344-2.pdf">Manuscript</a>, <a href="https://open.win.ox.ac.uk/pages/douaud/ukb-lifo-flica/">Code</a>, <a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-46344-2/MediaObjects/41467_2024_46344_MOESM1_ESM.pdf">Supplementary material 1</a>,
	<a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-46344-2/MediaObjects/41467_2024_46344_MOESM3_ESM.pdf">2</a>,
	<a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-46344-2/MediaObjects/41467_2024_46344_MOESM4_ESM.xlsx">3</a>,
	<a href="https://wapo.st/444Vtly">Washington Post article</a>,
	<a href="https://www.newsweek.com/scientists-identify-main-risk-factor-brain-aging-dementia-1883769">Newsweek article</a>, <a href="https://www.sfu.ca/research/performance-excellence/scholarly-impact-of-the-week/scholarly-impact-stories/2025/01/how-lifestyle-factors-and-genetics-affect-brain-health.html">SFU's Scholarly Impact of the Week</a> & <a href="https://www.ox.ac.uk/news/2024-03-27-risk-factors-faster-aging-brain-revealed-new-study">University of Oxford News article</a></i>
</li>

<li>
<u>E. Garg</u>,
P. Arguello-Pascualli,
<u>O. Vishnyakova</u>,
A.R. Halevy,
S. Yoo,
J.D. Brooks,
S.B. Bull,
F. Gagnon,
C.M.T. Greenwood,
R.J. Hung,
J.F. Lawless,
J. Lerner-Ellis,
J.K. Dennis,
R.J.S. Abraham,
J.-M. Garant,
B. Thiruvahindrapuram,
S.J.M. Jones,
CGEn HostSeq Initiative,
L.J. Strug,
A.D. Paterson,
L. Sun and
<u>L.T. Elliott</u>.
<em><a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1011192">
Canadian COVID-19 host genetics cohort replicates known severity associations</a>.</em>
2024. 
PLOS Genetics. 20(3). <i><a href="https://journals.plos.org/plosgenetics/article/file?id=10.1371/journal.pgen.1011192&type=printable">Manuscript</a>,
				 <a href="https://github.com/eg-r/HostSeq">Code</a> & <a href="https://www.cgen.ca/wp-content/uploads/2024/09/Annual-Report-2023-24-CGEn.pdf">Mention in the CGEn Annual Report 2023-24</a></i>
</li>
	
	<li>
		The COVID-19 Host Genetics Initiative.
		<em><a href="https://www.nature.com/articles/s41586-023-06355-3">
A second update on mapping the human genetic architecture of COVID-19.
		</a></em>
			2023.
			Nature.
			621(7977). pE7-26. <i><a href="https://www.nature.com/articles/s41586-023-06355-3.pdf">Manuscript</a>, <a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06355-3/MediaObjects/41586_2023_6355_MOESM1_ESM.pdf">Supplementary material 1</a>, <a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06355-3/MediaObjects/41586_2023_6355_MOESM3_ESM.pdf">2</a>, <a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06355-3/MediaObjects/41586_2023_6355_MOESM4_ESM.xlsx">3</a>, <a href="https://app.covid19hg.org">Results browser</a>, <a href="https://www.jpost.com/health-and-wellness/coronavirus/article-726607">Jerusalem Post article</a> & <a href="https://www.medrxiv.org/content/10.1101/2022.12.24.22283874v2">Preprint version</a></i>
	</li>
	
	<li> <u>O. Vishnyakova</u>, X. Song, K. Rockwood, <u>L.T. Elliott</u> and A. Brooks-Wilson. 
	<em><a href="https://link.springer.com/article/10.1007/s11357-023-00895-2">Physiological phenotypes have optimal values relevant to healthy aging: sweet spots deduced from the Canadian Longitudinal Study on Aging</a></em>.
		2023. GeroScience. 46(2). p1589-605. <i><a href="https://link.springer.com/content/pdf/10.1007/s11357-023-00895-2.pdf">Manuscript</a>, <a href="https://static-content.springer.com/esm/art%3A10.1007%2Fs11357-023-00895-2/MediaObjects/11357_2023_895_MOESM1_ESM.docx">Supplementary material</a>, <a href="https://vancouversun.com/video/e63a5320-d253-11ee-8a62-92ec5b55a028/conversations-that-matter-the-fountain-of-youth">Vancouver Sun interview</a> & <a href="https://www.sfu.ca/sfunews/stories/2024/01/sfu-researchers-zero-in-on-what-makes-some-people-healthier-than.html">SFU News article</a></i>
	
	
	</li>
	
	<li>
	J. Schneider, D. Stenning and <u>L.T. Elliott</u>.
	<em><a href="https://www.frontiersin.org/articles/10.3389/fspas.2023.1197358/full">Efficient galaxy classification through pretraining</a></em>. 2023.
	Frontiers in Astronomy and Space Sciences. 10. <i><a href="Schneider2023.zip">Manuscript</a>, <a href="https://github.com/jsa378/01_masters">Code</a> & <a href="https://summit.sfu.ca/item/36288">Thesis</a></i>
	</li>
	
	<li>
		S. Yoo, <u>E. Garg</u> et al. 
                <a href="https://link.springer.com/article/10.1186/s12863-023-01128-3">
                    <em>HostSeq: a Canadian whole genome sequencing and clinical data resource</em></a>. 2023. BMC Genomic Data. 24(26). <i><a href="https://link.springer.com/content/pdf/10.1186/s12863-023-01128-3.pdf">Manuscript</a>, <a href="https://link.springer.com/content/pdf/10.1186/s12863-023-01128-3.pdf">Supplementary material</a>, <a href="https://www.covid19immunitytaskforce.ca/hostseq-a-canadian-consortium-collecting-genetic-data-to-identify-risk-factors-for-covid-19-disease-and-health-outcomes/">COVID-19 Immunity Task Force summary</a> & <a href="https://www.cgen.ca/daco-main">Data</a></i></li>
      
	
	
            <li>
		    <u>S. Wang</u>,
		    <u>S. Ge</u>,
		    B. Sobkowiak,
		    L. Wang,
		    L. Grandjean,
		    C. Colijn and
		    <u>L.T. Elliott</u>.
                
                    <a href="https://www.liebertpub.com/doi/epub/10.1089/cmb.2022.0067"><em>Genome-wide association with uncertainty in the genetic similarity matrix</em></a>. 2023. Journal of Computational Biology. 30(2). 117-243.
		    <i><a href="Wang2023.pdf">Manuscript</a> & <a href="https://github.com/shijiaw/LMMTree">Code</a></i>
         
            </li>
	
	
		
	<li>
		<u>S. Ge</u>, <u>S. Wang</u> and <u>L.T. Elliott</u>.
		<a href=
		   "https://link.springer.com/article/10.1007/s11222-022-10170-7"><em>Shape modeling with spline partitions</em></a>. 2023. Statistics and Computing. 33(13). <i><a href="https://static-content.springer.com/esm/art%3A10.1007%2Fs11222-022-10170-7/MediaObjects/11222_2022_10170_MOESM1_ESM.pdf">Supplementary material</a> & <a href="https://github.com/ShufeiGe/Shape-Modeling-with-Spline-Partitions">Code</a></i>	</li>
	
	
	<li>
		S. Lee, J.D. Bijsterbosch F. Alfaro Almagro, <u>L.T. Elliott</u>, P. McCarthy, B. Taschler, R. Sala-Llonche, C.F. Beckmann, E.P. Duff, S.M. Smith and G. Douaud.		
		<em><a href="https://www.sciencedirect.com/science/article/pii/S1053811922009004">Amplitudes of resting-state functional networks&mdash;investigation into their correlates and biophysical properties</a></em>.
		2022. NeuroImage. 265. <i><a href="Lee2022.pdf">Manuscript</a>, <a href="https://ars.els-cdn.com/content/image/1-s2.0-S1053811922009004-mmc1.docx">Supplementary material 1</a> & <a href="https://ars.els-cdn.com/content/image/1-s2.0-S1053811922009004-mmc2.xlsx"> 2</a></i>
		
	</li>
	<li>
	
                <u>M.R.P. Parker</u>, L.L.E. Cowen, J. Cao and <u>L.T. Elliott</u>.
                
                    <em><a href="https://link.springer.com/article/10.1007/s13253-022-00509-y">Computational efficiency and precision for
replicated-count and batch-marked hidden population
			    models</em></a>. 2022. Journal of Agricultural, Biological and Environmental Statistics. 28(1). p43-58. <i><a href="Parker2022b.pdf">Manuscript</a>, <a href="https://static-content.springer.com/esm/art%3A10.1007%2Fs13253-022-00509-y/MediaObjects/13253_2022_509_MOESM1_ESM.r">Supplementary material 1</a>, <a href="https://static-content.springer.com/esm/art%3A10.1007%2Fs13253-022-00509-y/MediaObjects/13253_2022_509_MOESM2_ESM.r">2</a>, <a href="https://static-content.springer.com/esm/art%3A10.1007%2Fs13253-022-00509-y/MediaObjects/13253_2022_509_MOESM3_ESM.r">3</a> &
			    <a href="https://cran.r-project.org/web/packages/quickNmix/index.html">CRAN package</a></i>
         
            </li>
	
	
            <li>
                L. Wang,
		    <u>J. Min</u>,
		    R. Doig,
                <u>L.T. Elliott</u> and C. Colijn.
                
                    <em><a href="https://onlinelibrary.wiley.com/doi/full/10.1002/cjs.11722">Estimation of SARS-CoV-2 antibody prevalence through integration of serology and incidence data</em></a>. 2022. Canadian Journal of Statistics. 50(3). p734-50. <i><a href="Wang2022.pdf">Manuscript</a> & <a href="https://onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fcjs.11722&file=cjs11722-sup-0001-supinfo.pdf">Supplementary material</a></i
         
            </li>
	
	
	
	
	
            <li>
                C. Wang, A.B. Martins-Bach, F. Alfaro-Almagro, G. Douaud, J.C. Klein, A. Llera, C. Fiscone, R. Bowtell, <u>L.T. Elliott</u>, S.M. Smith, B.C. Tendler and K.L. Miller. 
                <a href="https://www.nature.com/articles/s41593-022-01074-w">
                    <em>Phenotypic and genetic associations of quantitative magnetic susceptibility in UK Biobank brain imaging</em></a>. 2022. Nature Neuroscience. 25(5583). p818-31.
		    <i><a href="https://www.nature.com/articles/s41593-022-01074-w.pdf">Manuscript</a>,
		    <a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-022-01074-w/MediaObjects/41593_2022_1074_MOESM1_ESM.pdf">Supplementary material 1</a> &
		    <a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-022-01074-w/MediaObjects/41593_2022_1074_MOESM3_ESM.xlsx">2</a></i>
	
            </li>
	
	
	<li>
		<u>J. Min</u> and <u>L.T. Elliott</u>. <a href="https://arxiv.org/abs/2204.03771"><em>Q-learning with online random forests</em></a>. 2022. 
		arxiv preprint 2204.03771. <i><a href="https://arxiv.org/pdf/2204.03771">Manuscript</a>, <a href="Min2022-supmat.pdf">Supplementary material</a> & <a href="https://summit.sfu.ca/item/34704">Thesis</a></i>
	</li>
	
	<li>
		<u>M.R.P. Parker</u>, Y. Li, <u>L.T. Elliott</u>, J. Ma and L.L.E. Cowen. <em><a href="https://onlinelibrary.wiley.com/doi/10.1002/cjs.11664">Under-reporting of COVID-19 in the Northern Health Authority region of British Columbia</a></em>.
		2021. Canadian Journal of Statistics. 49(4). p1018-38. <i><a href="Parker2021.pdf">Manuscript</a>, <a href="Parker2021-supmat.zip">Supplementary material</a> & <a href="http://web.archive.org/web/20211105231025/https://www.cbc.ca/listen/live-radio/1-109-daybreak-north/clip/15876862-b.c.-likely-underreporting-covid-cases-statistical-analysis-finds">CBC Radio One (Prince George) interview</a></i>
	</li>
	            <li>
                J.E. Stockdale, R. Doig, <u>J. Min</u>, N. Mulberry, L. Wang, <u>L.T. Elliott</u> and C. Colijn.
                <a href="https://www.eurosurveillance.org/content/10.2807/1560-7917.ES.2021.26.40.2001204">
                    <em>Long time frames to detect the impact of changing COVID-19 measures, Canada, March to July 2020</em></a>. 2021. Eurosurveillance. 26(4). <i><a href="https://github.com/jessicastockdale/LongTimeFrames">Code</a> & <a href="https://www.eurosurveillance.org/deliver/fulltext/eurosurveillance/26/40/20-01204_STOCKDALE_Supplementary_material.pdf?itemId=%2Fcontent%2Fsuppdata%2F10.2807%2F1560-7917.ES.2021.26.40.2001204-1&mimeType=pdf">Supplementary material</a></i></li>

	
            <li>
                <u>W. Chen</u> and
                <u>L.T. Elliott</u>.
                <a href="https://www.worldscientific.com/doi/10.1142/S0219720021500268">
                    <em>Compression for population genetic data through finite-state entropy</em></a>. 2021. Journal of Bioinformatics and Computational Biology. 19(5).
             <i>
		     <a href="Chen2021.pdf">Manuscript</a>, <a href="https://www.worldscientific.com/doi/suppl/10.1142/S0219720021500268">Supplementary material</a>, <a href="https://summit.sfu.ca/item/35057"><i>Thesis</i></a> & <a href="https://agent.engineering"><i>Website</i></a>
		    </i>
            </li>
	
	
            <li>
                <u>S. Wang</u>, <u>S. Ge</u>, C. Colijn, P. Biller, L. Wang and <u>L.T. Elliott</u>.
                
                <a href="https://www.liebertpub.com/doi/full/10.1089/cmb.2020.0375">
                    <em>Estimating genetic similarity matrices using phylogenies</em></a>. 2021. Journal of Computational Biology. 28(6). p587–600. <i><a href="Wang2021.pdf">Manuscript</a>, <a href="https://github.com/shijiaw/Expected-Genetic-Similarity-Matrices">Code</a> &
		    <a href="https://www.liebertpub.com/doi/suppl/10.1089/cmb.2020.0375/suppl_file/Supp_Data.pdf">Supplementary material</a></i>
            </li>
	
	
	
            <li>
                S.M. Smith, G. Douaud, <u>W. Chen</u>, T. Hanayik, F. Alfaro-Almagro, K. Sharp and
                <u>L.T. Elliott</u>.
                <a href="https://www.nature.com/articles/s41593-021-00826-4">
                    <em>An expanded set of genome-wide association studies of brain imaging phenotypes in UK Biobank</em></a>. 2021. Nature Neuroscience. 24(5). p737-45.
                <i><a href="https://open.win.ox.ac.uk/ukbiobank/big40/">Data</a>, <a href="https://www.ebi.ac.uk/gwas/publications/33875891">EBI Accessions</a> & <a href="https://www.sfu.ca/research/scholarly-impacts/uk-biobank-data-advances-human-genetics">SFU Scholarly Impact of the Week</a></i>
            </li>
	
	
	
            <li>
                <u>L.T. Elliott</u>. 
                <a href="https://www.liebertpub.com/doi/pdf/10.1089/cmb.2019.0440">
                    <i>Kinship solutions for partially observed multiphenotype data</i></a>. 2020. Journal of Computational Biology. 27(9). p1461-70.
                
                    <i><a href="https://github.com/lell/kgen">Code</a> &amp; <a href="https://www.liebertpub.com/doi/suppl/10.1089/cmb.2019.0440/suppl_file/Supp_Data.zip">Supplementary material</i>
                </a>
            </li>
            <li>
                N.J. Armstrong et al. 
                <a href="https://www.ahajournals.org/doi/abs/10.1161/STROKEAHA.119.027544">
                    <em>Common genetic variation indicates separate causes for periventricular and deep white matter hyperintensities</em></a>. 2020. Stroke. 51(7). p2111-21.
	    <i><a href="Armstrong2020.pdf">Manuscript</a>, <a href="https://www.ahajournals.org/doi/suppl/10.1161/STROKEAHA.119.027544/suppl_file/str_stroke-2019-027544_supp1.pdf">Supplementary material 1</a> & <a href="https://www.ahajournals.org/doi/suppl/10.1161/STROKEAHA.119.027544/suppl_file/str_stroke-2019-027544_supp2.pdf">2</a></i></li>
            <li>
                S.M. Smith, 
                <u>L.T. Elliott</u>, F. Alfaro-Almagro, P. McCarthy, T.E. Nichols, G. Douaud and K.L. Miller. 
                <a href="https://elifesciences.org/articles/52677">
                    <em>Brain aging comprises many modes of structural and functional change with distinct genetic and biophysical associations</em></a>. 2020. eLife. doi:10.7554/eLife.52677. 
                <i>
                    <a href="https://www.fmrib.ox.ac.uk/ukbiobank/BrainAgingModes/">Data</a> & <a href="https://www.fmrib.ox.ac.uk/ukbiobank/BrainAgingModes/BrainAgeModesCodes.tar.gz">Code</a>
                </i>
            </li>
            <li>
                <u>S. Ge</u>, <u>S. Wang</u>, Y.W. Teh, L. Wang and 
                <u>L.T. Elliott</u>. 
		    
		       <!--div style="float: right; border: 15px; border-color: #58525e; margin: 0px; padding:0px; text-align: center;">
					<a href="https://proceedings.neurips.cc/paper_files/paper/2019/file/043c2ec6c6390dd0ac5519190a57c88c-Paper.pdf"> 
						<img src="out.png" height=125 style="border: 5px; border-color: white; margin: 0px; padding: 0px;">
					</a>
	    </div-->
		    
                <o>
                    <a href="http://papers.nips.cc/paper/9153-random-tessellation-forests">Random tessellation forests</a></em>. 2019. Proceedings of the 33rd Conference on Neural Information Processing Systems. 
                <i><a href="https://proceedings.neurips.cc/paper_files/paper/2019/file/043c2ec6c6390dd0ac5519190a57c88c-Paper.pdf">Manuscript</a>, <a href="https://mondrian-shapes.github.io">Artwork</a>,
		    <a href="https://arxiv.org/src/1906.05440v5/anc/tess19">Code</a>, <a href="https://arxiv.org/src/1906.05440v5/anc/appendices.pdf">Supplementary material</a> & <a href="https://arxiv.org/abs/1906.05440">Preprint version</a></i>
                        
            <li>
                J. Mollink, S.M. Smith, 
                <u>L.T. Elliott</u>, M. Kleinnijenhuis, M. Hiemstra,  F. Alfaro-Almagro, J. Marchini, A.-M. van Cappellen van Walsum, S. Jbabdi and K.L. Miller.          
                <a href="https://www.nature.com/articles/s41593-019-0379-2">
                    <em>The spatial correspondence and genetic influence of interhemispheric connectivity with white matter microstructure</em></a>.
                          2019. Nature Neuroscience. 22(5). p809-19. <i><a href="Mollink2019.pdf">Manuscript</a> & <a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_MOESM1_ESM.pdf">Supplementary material</a></i>
	    </li>
                        
            <li>
                <u>L.T. Elliott</u>, M. De Iorio,  S. Favaro, K. Adhikari and Y.W. Teh.
                          
                <a href="https://projecteuclid.org/euclid.ba/1526695351">
                    <em>Modeling population structure under hierarchical Dirichlet processes</em></a>.
                          2019. Bayesian Analysis. 14(2). p313-39. 
                <i><a href="https://github.com/BigBayes/HDPStructure">
                    Code</a> &
			<a href="https://projecteuclid.org/journals/supplementalcontent/10.1214/17-BA1093/suppdf_1.pdf">Supplementary material</a></i>
            <li>
                <u>L.T. Elliott</u>, K. Sharp, F. Alfaro-Almagro, S. Shi, K.L. Miller, G. Douaud, J. Marchini and S.M. Smith. 
                <i>
                    <a href="http://www.nature.com/articles/s41586-018-0571-7">Genome-wide association studies of brain imaging phenotypes in UK Biobank</a></i>. 2018. Nature. 562(7726). p210-6. 
                <i><a href="https://open.win.ox.ac.uk/ukbiobank/big40/BIGv2/">Data</a>, 
		  <a href="https://www.fmrib.ox.ac.uk/ukbiobank/gwaspaper/index.html">Code</a>,
			<a href="winner.r" download>Winner's curse correction</a> &
			<a href="https://www.wired.com/story/uk-biobank-genetic-studies-healthcare/">Wired article</a></i>
            <li>
                C. Bycroft, C. Freeman, D. Petkova, G. Band,  
                <u>L.T. Elliott</u>, K. Sharp,  A. Motyer, D. Vukcevic, O. Delaneau, J. O&rsquo;Connell, A. Cortes, S. Welsh, A. Young, M. Effingham, G. McVean, S. Leslie, N. Allen, P. Donnelly and J. Marchini. 
                <i><a href="http://www.nature.com/articles/s41586-018-0579-z">The UK Biobank resource with deep phenotyping and genomic data</a></i>. 2018. Nature. 562(7726). p203-9.
		    <i><a href="https://www.ukbiobank.ac.uk">Data</a>, <a href="https://www.nature.com/articles/d41586-018-06948-3">Nature News and Views article</a>,
			    <a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-018-0579-z/MediaObjects/41586_2018_579_MOESM1_ESM.pdf">Supplementary material</a> &
		    <a href="https://www.biorxiv.org/content/10.1101/166298v1">Preprint version</a></i>
		</li>
                   
            <li>
		    S. Schwab, R. Harbord, V. Zerbi, <u>L.T. Elliott</u>, S. Afyouni, J.Q. Smith, M.W. Woolrich, S.M. Smith and T.E. Nichols. 
                <i>
                    <a href="https://www.sciencedirect.com/science/article/pii/S1053811918302842">Directed functional connectivity using dynamic graphical models</a></i>. 2018. NeuroImage. 175. p340-53
	    </li>
		    <li>
			    <u>L.T. Elliott</u>. <a href="https://discovery.ucl.ac.uk/id/eprint/1474452/"><i>Bayesian nonparametric models of genetic variation</i></a>. Doctoral thesis, University College London. 2016. <a href="algorithm_41" download><i>Mathematica code</i></a>
		    </li>

			<li>
				<u>L.T. Elliott</u> and Y.W. Teh. <em><a href="https://projecteuclid.org/journalArticle/Download?urlId=10.1214%2F16-EJS1197">A nonparametric HMM for genetic
imputation and coalescent inference</a>.</em> 2016. Electronic Journal of Statistics. 10(2). p3425-51. <em><a href="https://arxiv.org/abs/1611.00544">Preprint version</a> & <a href="https://github.com/lell/BNPPhase">Code</a></em>
			</li>
<li>
	R.M. Cichy, P. Sterzer, J. Heinzle, <u>L.T. Elliott</u> and F. Ramirez, J.-D. Haynes. <i><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/hbm.22020">Probing principles of large-scale object representation: Category preference and location encoding</a></i>. 2013. Human Brain Mapping. 34(7). p1636-51. <a href="https://onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fhbm.22020&file=HBM_22020_sm_SuppInfo.doc"><i>Supplementary material</i></a>
</li>
	<li><u>L.T. Elliott</u> and Y.W. Teh. <a href="https://papers.nips.cc/paper_files/paper/2012/file/84438b7aae55a0638073ef798e50b4ef-Paper.pdf"><i>Scalable imputation of genetic data with a discrete fragmentation-coagulation process</i></a>. 2012. Proceedings of the 25th Conference on Neural Information Processing Systems. <a href="https://github.com/lell/BNPPhase"><i>Code</i></a>
	</li>	
	
	<li>
	I. Murray and <u>L.T. Elliott</u>.
		<a href="https://arxiv.org/abs/1204.3187"><i>Driving Markov chain Monte Carlo with a dependent random stream</a></i>. 2012. arxiv preprint 1204.3187.
			<i>
				<a href="https://arxiv.org/pdf/1204.3187">Manuscript</a>,
				<a href="https://arxiv.org/src/1204.3187v1/anc">Code</a>,
				<a href="https://radfordneal.wordpress.com/2012/05/03/non-random-mcmc/">Mention in Radford Neal's blog</a>
			</i>
</li>
<li>
Y.W. Teh, C. Blundell and <u>L.T. Elliott</u>.
<a href="https://papers.nips.cc/paper_files/paper/2011/file/5e388103a391daabe3de1d76a6739ccd-Paper.pdf"><i>Modelling genetic variations using fragmentation-coagulation processes</i></a>. 2011. Proceedings of the 24th Conference on Neural Information Processing Systems
</li>
	
	<li>
		Y. Chen, P. Namburi, <u>L.T. Elliott</u>, J. Heinzle, C.S Soon, M.W.L. Chee and J.-D. Haynes. <i><a href="https://www.sciencedirect.com/science/article/abs/pii/S1053811910010086">Cortical surface-based searchlight decoding</a></i>. 2011. NeuroImage. 56(2). p582-92.
		<i><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1053811910010086-mmc1.doc">Supplementary material</a></i>
	</li>

	</ul>
</p>
</div>

</div>
</div>
<div class="footer" >
<span>&nbsp;
	<div class="bottom" style="bottom:0px; position:relative;">
<i>Simon Fraser University acknowledges the unceded Traditional Coast Salish Lands including the Tsleil-Waututh (s&#601;l&#787;ilw&#787;&#601;ta&#660;&#620;), Kwikwetlem (k&#695;ik&#695;&#601;&#411;&#787;&#601;m), Squamish (S&#7733;wx&#817;w&uacute;7mesh &Uacute;xwumixw) and Musqueam (x&#695;m&#601;&theta;k&#695;&#601;y&#787;&#601;m) Nations.</i>
</div>&nbsp;
</span>
</div>

<script>
$( "#button" ).button();
$( "#button-icon" ).button({
	icon: "ui-icon-gear",
	showLabel: false
});

$( "#radioset" ).buttonset();

$( "#controlgroup" ).controlgroup();

$( "#dialog-link, #icons li" ).hover(
	function() {
		$( this ).addClass( "ui-state-hover" );
	},
	function() {
		$( this ).removeClass( "ui-state-hover" );
	}
);
</script>
</span>
</div>
</body>
</html>
